pages/about/research/recruitment/general-fed.md

```diff
@@ -0,0 +1,53 @@
+---
+permalink: /about/research/recruitment/general-fed/
+layout: styleguide
+title: How the federal USWDS community can participate in our research
+category: About
+lead: Help shape the future of U.S. government websites.
+redirect_from:
+  - /research/recruitment/federal/
+changelog:
+  key: about-research-recruitment-federal
+subnav: true
+---
+
+How the federal USWDS community can participate in our research
```

Comment:
Please remove this line. It's duplicating the page title.

Reply:
Removed in eb51406. Thanks!